.vscode/launch.json

```diff
@@ -24,16 +25,16 @@
             "type" : "go",
             "request": "launch",
             "mode" : "auto",
-            "program": "${workspaceFolder}/cmd/sqlcmd",
-            "args" : ["-Q", "EXIT(select 100 as Count)"],
+            "program": "${workspaceFolder}/cmd/modern",
+            "args" : ["-Q", "EXIT(select network_protocol from sys.dm_exec_sessions)", "-S", "lpc:."],
         },
         {
             "name" : "Run file query",
             "type" : "go",
             "request": "launch",
             "mode" : "auto",
-            "program": "${workspaceFolder}/cmd/sqlcmd",
-            "args" : ["-i", "testdata\\select100.sql"],
+            "program": "${workspaceFolder}/cmd/modern",
+            "args" : ["-S", "np:.", "-i", "${workspaceFolder}/cmd/sqlcmd/testdata/select100.sql"],
```

Comment:
Do we need to turn this into unit test?

Reply:
the tests exist. The launch.json is needed to debug the full app starting in main.go